picoquicfirst/picoquicdemo.c

```diff
@@ -293,6 +294,16 @@ int client_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode,
             fprintf(stdout, "Waiting for packets.\n");
             break;
         case picoquic_packet_loop_after_receive:
+            if (cb_ctx->cnx_client->is_1rtt_acked) {
+                // TODO
+                uint64_t simulated_time = 0;
```

Comment:
I'm pretty sure `simulated_time` here is not correct.

Reply:
What do you use it for? The socket loop is designed to use real time, not simulated time. The "current_time" variable normally tracks that.